docs/advanced_topics/icons.rst

```diff
@@ -0,0 +1,85 @@
+.. _icons:
+
+=====
+Icons
+=====
+
+Wagtail comes with an icon set.
+The icons are used throughout the admin interface.
+
+Elements that use icons are:
+
+- :ref:`Various menu items <admin_hooks>`
+- :ref:`ModelAdmin menu items <modeladmin_menu_icon>`
+- :ref:`Streamfield blocks <basic_block_type_icon>`
+- :ref:`Rich text editor toolbar buttons <extending_the_draftail_editor>`
+- ...
+
+This document describes how to choose and add icons.
+
+Available icons and their names
+-------------------------------
+
+Enable the :ref:`styleguide <styleguide>` to view icons and their names.
+
+Alternatively inspect the source code ``wagtail/admin/wagtail_hooks.py``.
+The filename without ``.svg`` is the icon name.
+
+Add a custom icon
+-----------------
+
+Draw or download an icon.
+
+The SVG should:
+
+ - Have a size of 16x16 points
```

Comment:
It is unclear to me what 16x16 points means. Is it the viewbox?

Reply:
Yes. I added an example.

Reply:
Cheers @allcaps!
I feel this may prove tricky for non-SVG proficient developers who just want to use a FontAwesome icon (e.g. https://fontawesome.com/icons/clipboard-list has a viewbox of `0 0 384 512` (and I, for example don't have the tools to convert it to a `0 0 16 16` viewbox. Changing the value ends up with a blank icon)

Reply:
I'm not sure the same size matters that much as the final size is defined in CSS. 
Maybe the ratio is significant. But I'm not sure about that too. I have to experiment.

Reply:
aside from this line which is confusing, the rest LGTM and ready to be merged on my opinion

Reply:
I am experiencing the same issue. Trying to use fork-awesome icons which look like this:
```svg
<svg width="1536" height="1536" xmlns="http://www.w3.org/2000/svg">
  <path d="M1401 1547l-6 6c-75 ... " />
</svg>
```

Simply removing the width, height, and xmlns attributes and adding `viewbox="0 0 16 16"` does not scale it correctly. I'm not sure what kind of tools or attributes should be used to convert an existing SVG into a wagtail-flavored SVG.